### PR TITLE
Atoms/p6 calendar

### LINKED ISSUE #116: Create a timepicker Web component

### PR DESCRIPTION
**Pull request by jswale**
Fixes #116 #115

### CODE REVIEW

**Comment by vincentv** [comment]
please, format the source code.

**Comment by jswale** [comment]
> please, format the source code.

I thought I did it...

File: src/utils/attribute.ts [hunk 1]
```diff
@@ -8,6 +8,13 @@ export function isEmpty(value: unknown): boolean {
   );
 }
 
+export function convertAsDate(value: string | undefined): Date | undefined {
```
Comment (vincentv) [inline src/utils/attribute.ts:11]: toDate?

File: src/utils/translations.ts [hunk 2]
```diff
@@ -14,7 +14,9 @@ export interface L10n {
  * Falls back to english if no language is found.
  * @param element The element to find a lang attribute for.
  */
-function getClosestLanguage(element: HTMLElement = document.body): string {
+export function getClosestLanguage(
```
Comment (vincentv) [inline src/utils/translations.ts:17]: We could also look at the browser language?

File: src/components/atoms/p6-calendar/locales/p6-calendar.i18n.fr.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "cancelLabel": "Annuler",
+  "clearLabel": "Vider",
```
Comment (vincentv) [inline src/components/atoms/p6-calendar/locales/p6-calendar.i18n.fr.json:3]: "Effacer" instead of "Vider"

**Pull request merged**